ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -106,12 +109,16 @@ open class ContactView: UIView {
             constraints.append(contentsOf: identifierLayoutConstraints())
         }
         titleLabel.translatesAutoresizingMaskIntoConstraints = false
-        labelContainer.addSubview(titleLabel)
+
+        pressedStateOverlay.translatesAutoresizingMaskIntoConstraints = false
+        avatarView.addSubview(pressedStateOverlay)
+        constraints.append(contentsOf: pressedStateOverlayLayoutConstraints())
```

Comment:
nit: would be cleaner to just pass the array of constraints here instead of having a separate method in this case, especially considering that the method only gets called once.